public/app/features/alerting/unified/components/DynamicTable.tsx

```diff
@@ -84,7 +84,7 @@ export const DynamicTable: FC<DynamicTableProps> = ({
   );
 };
 
-const getStyles = (cols: DynamicTableColumnProps[], isExpandable: boolean) => {
+const getStyles = <T extends any>(cols: Array<DynamicTableColumnProps<T>>, isExpandable: boolean) => {
```

Comment:
It's not the biggest thing (as `T` is inferred), but I think its better to use `extends unknown` instead of `any` to keep that `any` counter down 😌